docs/Documentation/run_RBM.md

```diff
@@ -89,7 +87,7 @@ This will create the executable,
 
 For this example, copy **rout** to the folder **Test_Case**
 
-Create control file, **salmon.inp_DA**, to run the modified Lohmann routing model using the file, **Rout.Cells.init**, the first time and the file, **Rout.Cells**, if the routing model, **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold parameters. The routing model requires a file with unit hydrographs for each cell, or, in this case, a file, **UH_ALL**, with a single unit hydrograph that is applied to all grid cells.
+Create control file **salmon.inp_DA** to run the modified Lohmann routing model using the file **Rout.Cells.init** the first time or the file **Rout.Cells** if the routing model **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold parameters. The routing model requires a file with unit hydrographs for each cell, or, in this case, a file, **UH_ALL**, with a single unit hydrograph that is applied to all grid cells.
```

Comment:
Also, after `**Rout.Cells.init** have `(for the first time)` with the parenthesis for a bit more clarity.